src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let action = 0; action < actions.length; action++) {
```

Comment:
It's not an `action` really, it's just an iterator's temp variable. So you can name it just `i` for example.